src/blockchains/erc20/erc20_worker.ts

```diff
@@ -59,7 +59,7 @@ export class ERC20Worker extends BaseWorker {
     this.allOldContracts = [];
   }
 
-  async init(exporter?: Exporter) {
+  async init(storage: KafkaStorage | Map<string, KafkaStorage>) {
```

Comment:
Workers receive information for which output Kafka topics need to be filled and produce the needed data.